src/spatialdata_plot/pl/render.py

```diff
@@ -416,9 +449,15 @@ def _render_points(
                 y_ext = [min(y_ext[0], previous_ylim[1]), max(y_ext[1], previous_ylim[0])]
             else:  # case for e.g. labels
                 y_ext = [min(y_ext[0], previous_ylim[0]), max(y_ext[1], previous_ylim[1])]
```

Comment:
I'd consider bundling this code in a private function since it's duplicate from above.